component/exporter.go

```diff
@@ -149,30 +149,57 @@ type exporterFactory struct {
 }
 
 // WithTracesExporter overrides the default "error not supported" implementation for CreateTracesExporter.
+// Deprecated: [v0.55.0] Use WithTracesExporterAndStabilityLevel instead.
```

Comment:
Make this call into `WithTracesExporterAndStabilityLevel` with `Undefined` level?